src/test/java/org/jboss/xavier/analytics/functions/HelperFunctionsTest.java

```diff
@@ -36,6 +36,7 @@
                 {"Microsoft Windows XP Professional (32-bit)", false, false, true, false},
                 {"Enterprise", false, false, true, false},
                 {"", false, false, false, true},
+                {"Not detected", false, false, false, true},
```

Comment:
If we change the value of the constant in WorkloadInventoryReportModel will make the test fail.    
Shouldn't here be better to use `WorkloadInventoryReportModel.OS_NAME_DEFAULT_VALUE` as it's this what we are setting in the code ?

Reply:
Done